objects/obj_popup/Draw_0.gml

```diff
@@ -138,8 +146,8 @@ if ((zm=0) and (type<=4)) or (type=98){
     
     // if (image!="") then draw_text_ext(x1+126+150,y1+152,string(text),-1,384-150);
     // if (text2!="") then draw_text_ext(x1+126,y1+309,string(text2),-1,384);
-    
-    var img;img=-1;
+    // TODO change this into an array in a function (like romanNumerals does in here)
```

Comment:
for a future PR?

Reply:
yes, basically would add even more work. My idea is to basically have a simple structure:
something like 
{
    enemies: {
        tau:{ 
            image: 1;
            enemy: 1;
            other stuff 
        }
    }
}

basically have a json with all the enemy data, and move into json structures rather than simple arrays or just strings. But that  is another PR imo